modules/cluster-logging-kibana-tolerations.adoc

```diff
@@ -1,60 +1,64 @@
 // Module included in the following assemblies:
 //
-// * logging/cluster-logging-visualizer.adoc
+// * logging/scheduling_resources/logging-taints-tolerations.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-kibana-tolerations_{context}"]
 = Using tolerations to control the log visualizer pod placement
 
-You can control the node where the log visualizer pod runs and prevent
-other workloads from using those nodes by using tolerations on the pods.
-
-You apply tolerations to the log visualizer pod through the `ClusterLogging` custom resource (CR)
-and apply taints to a node through the node specification. A taint on a node is a `key:value pair` that
-instructs the node to repel all pods that do not tolerate the taint. Using a specific `key:value` pair
-that is not on other pods ensures only the Kibana pod can run on that node.
+You can use a specific `key:value` pair that is not on other pods to ensure that only the Kibana pod can run on the specified node.
```

Comment:
Elsewhere in these sections, there are a few instances of "key/value" (no markup). And in the example command below in this module, there's `<key>=<value>` (with backticks). Suggest flipping the instance here to either:
```suggestion
You can use a specific key/value pair that is not on other pods to ensure that only the Kibana pod can run on the specified node.
```
(^ probably preferable for general usage)
or
```suggestion
You can use a specific `<key>=<value>` pair that is not on other pods to ensure that only the Kibana pod can run on the specified node.
```
(^ if you really want to keep the markup)